docs/dev.rst

```diff
@@ -60,6 +60,20 @@ Now, use the ``tutor dev ...`` command-line interface to manage the development
 
   Read more about bind-mounts :ref:`below <bind_mounts>`.
 
+Conserving system resources
+---------------------------
+
+When developing, you'll usually want the development environment to take up as little system resources as possible.  To do so, we recommend setting the following Tutor configuration and relaunching your environment::
+
+  tutor config save \
+    --set OPENEDX_CMS_CELERY_WORKERS=1 \
+    --set OPENEDX_LMS_CELERY_WORKERS=1 \
+    --set OPENEDX_CMS_UWSGI_WORKERS=1 \
+    --set OPENEDX_LMS_UWSGI_WORKERS=1 \
```

Comment:
I'd rather avoid asking users to manually set these values. Instead, we should automatically default to workers=1 in development. Can we do that? For instance by overriding the celery config in development?